File: CuponSmart/src/cuponsmart/modelo/pojo/respuesta/RespuestaPromocionSucursal.java
package cuponsmart.modelo.pojo.respuesta;

import java.util.List;
import cuponsmart.modelo.pojo.entidad.PromocionSucursal;

public class RespuestaPromocionSucursal extends Mensaje{
    private List<PromocionSucursal> contenido;
    
    public List<PromocionSucursal> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<PromocionSucursal> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaPromocionSucursal.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.PromocionSucursal

class RespuestaPromocionSucursal(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<PromocionSucursal>? = null
): Mensaje(error, mensaje)